backend/src/graphql/resolver/TransactionLinkResolver.test.ts

```diff
@@ -160,6 +171,56 @@ describe('TransactionLinkResolver', () => {
           ],
         })
       })
+
+      describe('after one day', () => {
+        beforeAll(async () => {
+          jest.useFakeTimers()
+          /* eslint-disable-next-line @typescript-eslint/no-empty-function */
+          setTimeout(() => {}, 1000 * 60 * 60 * 24)
+          jest.runAllTimers()
```

Comment:
What in detail do these three lines of code before?

Reply:
```javascript
// say the test suite that we want to use a faked clock
jest.useFakeTimers()
// a void function that should run after 24h
setTimeout(() => {}, 1000 * 60 * 60 * 24)
// let all timers, e.g. the void function above, run 
jest.runAllTimers()
// Now the clock is 24h later
```